### PR TITLE
Visual Studio Code docs

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [ ] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

**Review by pabloest — APPROVED** [review]

File: docs/extensions/overview.md [hunk 1]
```diff
@@ -11,14 +11,13 @@ import MoreHelp from "/src/components/MoreHelp"
 
 Several third-party tools include Semgrep extensions.
 
-
-### Editor
+## Editor
```
Comment (ajbt200128) [inline docs/extensions/overview.md:14]: We should probably split this up somehow, for example:

## Editor

### Official Extensions
- Microsoft Visual Studio Code: [`semgrep-vscode`](https://marketplace.visualstudio.com/items?itemName=semgrep.semgrep)
- Emacs: [`lsp-mode`](https://github.com/emacs-lsp/lsp-mode)

### 3rd party extensions

- IntelliJ IDEA: [`semgrep-idea-plugin`](https://github.com/jtmelton/semgrep-idea-plugin)
- Vim: [`semgrep.vim`](https://github.com/returntocorp/semgrep.vim)

**Review by ajbt200128 — CHANGES_REQUESTED** [review]
Looks good overall! Added one comment

File: docs/extensions/overview.md (hunk 1 above)
Comment (pabloest) [reply docs/extensions/overview.md:14]: For the latter, maybe “Community-contributed extensions?”

Comment (adamkvitek) [reply docs/extensions/overview.md:14]: Let's split them - I agree, to Community-contributed extensions - what do you think @ajbt200128? :)

File: docs/extensions/overview.md [hunk 2]
```diff
@@ -11,14 +11,13 @@ import MoreHelp from "/src/components/MoreHelp"
...
 
 - IntelliJ IDEA: [`semgrep-idea-plugin`](https://github.com/jtmelton/semgrep-idea-plugin)
 - Microsoft Visual Studio Code: [`semgrep-vscode`](https://marketplace.visualstudio.com/items?itemName=semgrep.semgrep)
 - Vim: [`semgrep.vim`](https://github.com/returntocorp/semgrep.vim)
```
Comment (adamkvitek) [inline docs/extensions/overview.md:18]:
```suggestion
## Editor

### Official Extensions

- Microsoft Visual Studio Code: [`semgrep-vscode`](https://marketplace.visualstudio.com/items?itemName=semgrep.semgrep)
- Emacs: [`lsp-mode`](https://github.com/emacs-lsp/lsp-mode)

### Community-contributed extensions

- IntelliJ IDEA: [`semgrep-idea-plugin`](https://github.com/jtmelton/semgrep-idea-plugin)
- Vim: [`semgrep.vim`](https://github.com/returntocorp/semgrep.vim)
```

**Review by ajbt200128 — APPROVED** [review]
LGTM

**Pull request merged**